tasks.py

```diff
@@ -75,6 +75,7 @@ class TargetHost:
     {
         "build01-dev": TargetHost(hostname="51.12.57.124", nixosconfig="build01"),
         "ghafhydra-dev": TargetHost(hostname="51.12.56.79", nixosconfig="ghafhydra"),
+        "binarycache": TargetHost(hostname="172.18.20.109", nixosconfig="binarycache"),
```

Comment:
Could we name the 'binarycache' alias to something that would indicate the environment it's going to be used in. Maybe 'binarycache-ficolo'?